src/pycountry/__init__.py

```diff
@@ -4,26 +4,25 @@
 import os.path
 import unicodedata
 
-import pycountry.db
+import importlib_metadata
+import importlib_resources
 
-try:
-    import pkg_resources
+import pycountry.db
 
-    resource_filename = pkg_resources.resource_filename
-except ImportError:
 
-    def resource_filename(package_or_requirement, resource_name):
-        return os.path.join(os.path.dirname(__file__), resource_name)
+def resource_filename(package_or_requirement, resource_name):
+    return str(importlib_resources.files(package_or_requirement) / resource_name)
```

Comment:
If we prioritise importing `importlib_resources` over `importlib.resources` (like you suggested in the other comment), then the function should always be available, because the backported package will be installed on versions that don't have the function.